src/Adyen/Service/ResourceModel/BinLookup/Get3dsAvailability.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Adyen\Service\ResourceModel\BinLookup;
+
+class Get3dsAvailability extends \Adyen\Service\AbstractResource
+{
+    /**
+     * @var string
+     */
+    protected $endpoint;
+
+    /**
+     * Include applicationInfo key in the request parameters
+     *
+     * @var bool
+     */
+    protected $allowApplicationInfo = true;
```

Comment:
could you put this to false as we do not support it for this endpoint.

Reply:
Sure!